cognite/client/data_classes/filters.py

```diff
@@ -655,9 +761,24 @@ class Search(FilterWithPropertyAndValue):
 
 
 class SpaceFilter(FilterWithPropertyAndValueList):
+    """Filters results based on the space.
```

Comment:
I believe this won't work outside of instances
```suggestion
    """Filters instances based on the space.
```